lua/ulx/modules/sh/yeet.lua

```diff
@@ -0,0 +1,37 @@
+local CATEGORY_NAME = "Fun"
+
+local MAX_STRENGTH = 500000	
+local DEFAULT_STRENGTH = 6000 
+
+function yeetPlayer( caller, targets, strength )
+	
+    for _, v in pairs( targets ) do
+		
+    if not v:IsPlayer() then return end
+    if not v:Alive() then return end
+
+    if v:IsFrozen() then
+		
+   	    v:Freeze( false )
+					
+    end
+	
+    if v:GetMoveType() == MOVETYPE_NOCLIP then
+			
+        v:SetMoveType( MOVETYPE_WALK )
+			
+    end
+
+    v:SetVelocity( Vector( math.random( 50000 ) -20000, math.random( 50000 ) -20000, math.Clamp( strength, DEFAULT_STRENGTH, MAX_STRENGTH ) ) )
```

Comment:
instead of math.random(5000) -2000 couldn't you do
`math.random( -20000, 30000 )` or maybe it should be `math.random( -20000, 20000 )`

Reply:
i think clamping strength before the SetVelocity line would also make it easier to read
```lua
strength = math.Clamp( strength, DEFAULT_STRENGTH, MAX_STRENGTH )
v:SetVelocity( ... )